OpenEdXMobile/src/test/java/org/edx/mobile/test/http/ApiTests.java

```diff
@@ -318,6 +328,22 @@ public boolean apply(CourseComponent component) {
 
     }
 
+    @Test
+    public void testRemoteConfigResponse() throws Exception {
```

Comment:
Test cases are failing 😢

Reply:
It works on GitHub Action and my local machine as well. Can you please share the error details? 😿